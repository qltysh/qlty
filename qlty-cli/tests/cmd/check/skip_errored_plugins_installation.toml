bin.name = "qlty"
args = ["check", "--all", "--no-cache", "--skip-errored-plugins"]
