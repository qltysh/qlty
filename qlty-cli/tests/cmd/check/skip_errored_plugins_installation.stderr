WARNING: Unknown field 'plugins.definitions.dotenv-linter.output_missing' in linters/dotenv-linter/plugin.toml. This will be ignored.
WARNING: Unknown field 'plugins.definitions.ktlint.drivers.lint.cache' in linters/ktlint/plugin.toml. This will be ignored.
WARNING: Unknown field 'plugins.downloads.swiftlint.download_type' in linters/swiftlint/plugin.toml. This will be ignored.
WARNING: Unknown field 'plugins.downloads.swiftlint.github' in linters/swiftlint/plugin.toml. This will be ignored.
WARNING: Unknown field 'plugins.definitions.dotenv-linter.output_missing' in qlty.toml. This will be ignored.
WARNING: Unknown field 'plugins.definitions.ktlint.drivers.lint.cache' in qlty.toml. This will be ignored.
WARNING: Unknown field 'plugins.downloads.swiftlint.download_type' in qlty.toml. This will be ignored.
WARNING: Unknown field 'plugins.downloads.swiftlint.github' in qlty.toml. This will be ignored.
âœ” No issues
